core/feature_flag_list.py

```diff
@@ -210,7 +210,7 @@ class FeatureNames(enum.Enum):
         (
             'The flag allows voice artists to add voiceovers in a specific '
             'accent for the given language.',
-            feature_flag_domain.ServerMode.TEST
+            feature_flag_domain.ServerMode.PROD
```

Comment:
The feature is tested in the backup server by our QA testers.

Reply:
@Nik-09 There will be docs to prove this. Please link them here.

Reply:
Thank you Sean for pointing this, added here and in the description. 
https://docs.google.com/document/d/1v5Q6zxJiQEaSAM-pIEoapmJbd7htH2Dgf08xOmRgESA/edit

Reply:
@Nik-09, you can resolve this comment now.